packages/commonwealth/client/scripts/hooks/useTransactionHistory.ts

```diff
@@ -0,0 +1,51 @@
+import { trpc } from '../utils/trpcClient';
+import { buildEtherscanLink } from '../views/modals/ManageCommunityStakeModal/utils';
+import { FilterOptions } from '../views/pages/MyCommunityStake/types';
+
+export type TransactionHistoryProps = {
+  filterOptions: FilterOptions;
+  addressFilter: string[];
+};
+
+const useTransactionHistory = ({
+  filterOptions,
+  addressFilter,
+}: TransactionHistoryProps) => {
+  const { data } = trpc.community.getStakeTransaction.useQuery({
+    addresses: addressFilter.length > 0 ? addressFilter.join(',') : undefined,
+  });
+
+  const WEI_PER_ETHER = 1000000000000000000;
```

Comment:
should this be a global constant in procotol domain or just used here?

Reply:
Yep it is global, I will find some place shared file to put it